### DISCUSSION TITLE
Suggestions to the Owner

### DISCUSSION THREAD

**Discussion by soulcan** [aclap-dev/video-downloadhelper#32] [Help and Q&A] [answered]
### Your question

Hi all. Hope everybody is fine. Sorry, this won't be a question but one suggestion for now:  

Sometimes the videos start downloading, and for numerous reasons stop, I will have to start the download by going to history, finding the page and starting the video again. Is there a possibility that when a video stops downloading, there would be a link on the add-on that remains and then we can click it to download again?

Thanks for your understanding.
p.s.: Maybe here on this forum, there should be a discussion topic as "Suggestions"

All my best

### Addon & CoApp version and details

VdhCoApp 2.0.10
Video DownloadHelper Version 8.1.6.0
Browser locale: en-GB
Production build Built on 2024-01-18
Platform Mac x86-64 (Sonoma 14.3) Chrome/121.0.0.0 Safari/537.36

**Comment by jcfrog** [comment, marked as answer]
Hi,
Could you try the latest beta, there should be less interruptions : https://github.com/aclap-dev/video-downloadhelper/wiki/Beta-versions

**Reply by soulcan** [reply]
Thanks. Will try.

**Comment by paulrouget** [comment]
> there would be a link on the add-on that remains and then we can click it to download again?

Hey, good news, we are adding a "retry" button in version v8.3 (work in progress)!
